### PR TITLE
Add themes support

### PR DESCRIPTION
**Pull request by i-Living**

### CODE REVIEW

**Comment by kuzkok** [comment]
you should split big less files, they is unreadable.

**Comment by almostcake** [comment]
image
should be inline
```css
display: flex;
flex-direction: row;
```

**Comment by almostcake** [comment]
image
redundant scroll

**Comment by almostcake** [comment]
image
looks really ugly. I suppose that on the form (not in the dialog) this component must be full-height

**Comment by almostcake** [comment]
image
that button has strange style

**Comment by almostcake** [comment]
bbox block in dialog looks:
image
but on the form (metadata=>item)
image

and get capabilities button looks better. I mean that "non theme" styles should be removed. I think that these styles have appeared from Geoportal, yep?

File: addon/styles/themes/blue-sky/elements/button.overrides [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+/* Flexberry Rdit Layermap Segment */
+.flexberry-edit-layermap {
+    .ui.small.fluid.buttons:not(.basic):not(.inverted) {
+      &>.button.flexberry-button {
+          background: @whitecolor;
+          color: @uiIconButtonColor;
+          border: 1px solid @bluesky;
+          &.active {
+              background: @lightbluecolor !important;
```
Comment (almostcake) [inline addon/styles/themes/blue-sky/elements/button.overrides]: do you understand that it is already hardcoded selector, and you place important there?

File: addon/styles/themes/blue-sky/elements/button.overrides [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          }
+          &:hover {
+              opacity: 0.8;
+              color: @uiIconButtonColor !important;
```
Comment (almostcake) [inline addon/styles/themes/blue-sky/elements/button.overrides]: do you understand that it is already hardcoded selector, and you place important there?

File: addon/styles/themes/blue-sky/elements/button.overrides [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          }
+      }
+    }
+    .button.flexberry-button {
+        color: @uiIconButtonColor !important;
```
Comment (almostcake) [inline addon/styles/themes/blue-sky/elements/button.overrides]: do you understand that it is already hardcoded selector, and you place important there?

File: addon/styles/themes/blue-sky/elements/button.overrides [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        &:hover {
+          color: @white !important;
```
Comment (almostcake) [inline addon/styles/themes/blue-sky/elements/button.overrides]: do you understand that it is already hardcoded selector, and you place important there?

File: addon/styles/themes/blue-sky/modules/modal.overrides [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    .ui.tabular.menu .item {
+        color: @bluecolor;
+        &.active {
+            color: @darkbluecolor;
+        }
+        &:hover {
+            color: @bluecolor;
+        }
+        &.disabled i.icon:before {
+            color: #999 !important;
```
Comment (almostcake) [inline addon/styles/themes/blue-sky/modules/modal.overrides]: variable

File: addon/styles/themes/default/modules/sidebar.overrides [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+            border-radius: 0px 0px 5px 5px;
+        }
+    }
+    .ui.styled.accordion {
+        box-shadow: none;
+        .accordion {
+            margin: .5em 0;
+            border-radius: 5px 0px 0px 5px;
+            &:first-child .title {
+                border-top: 1px solid rgba(34, 36, 38, 0.15);
```
Comment (almostcake) [inline addon/styles/themes/default/modules/sidebar.overrides]: variable

**Pull request merged**